src/query.c

```diff
@@ -207,12 +207,14 @@ SDB_API char *sdb_querys(Sdb *r, char *buf, size_t len, const char *_cmd) {
 	quot = NULL;
 	json = NULL;
 	if (*p == '#') {
+		char buffer[256];
```

Comment:
the snprintf below can't fill 256 bytes, allocating 16 should be enough
```suggestion
		char buffer[16];
```